packages/yourloops/app/app.tsx

```diff
@@ -26,72 +26,51 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import * as React from 'react';
-import { render as renderDOM } from "react-dom";
-import { ThemeProvider } from "@material-ui/core/styles";
-import {
-  BrowserRouter as Router,
-  Switch,
-  Route,
-} from "react-router-dom";
-import bows from 'bows';
-
 import "@fontsource/roboto";
 import "branding/theme-base.css";
 import "branding/theme.css";
 
-import { initI18n } from "../lib/language";
-
-import { theme } from "../components/theme";
-import LoginPage from '../pages/login';
-import HcpPage from '../pages/hcp';
-import PatientPage from "../pages/patient";
-import { RequestPasswordResetPage, ConfirmPasswordResetPage } from '../pages/password-reset';
-import { AuthProvider } from '../lib/auth/hook/use-auth';
-import PrivateRoute from '../components/private-route';
-
-class Yourloops {
-  private log: Console;
+import * as React from "react";
 
-  constructor() {
-    this.log = bows("Yourloops");
-  }
-
-  public async init(): Promise<void> {
-    this.log.info("Init");
-
-    await initI18n();
-    this.log.debug("i18next initialized");
-  }
+import {
+  ConfirmPasswordResetPage,
+  RequestPasswordResetPage,
+} from "../pages/password-reset";
+import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
 
-  public render(): void {
-    const router = this.router();
-    let div = document.getElementById("app");
-    if (div === null) {
-      div = document.createElement("div");
-      div.id = "app";
-      document.body.appendChild(div);
-    }
-    renderDOM(router, div);
-  }
+import { AuthProvider } from "../lib/auth/hook/use-auth";
+import HcpPage from "../pages/hcp";
+import LoginPage from "../pages/login";
+import PatientPage from "../pages/patient";
+import PrivateRoute from "../components/private-route";
+import { Suspense } from "react";
+import { ThemeProvider } from "@material-ui/core/styles";
+import { theme } from "../components/theme";
 
-  private router(): JSX.Element {
-    return (
+const Yourloops: React.FunctionComponent = () => {
```

Comment:
Use a plain function, no need to declare a var as a function here

Reply:
Why do you prefer plain function over arrow function ?

Reply:
I think for "globale" module functions, the code the more readable.